.editorconfig

```diff
@@ -5,7 +5,16 @@ root = true
 
 # Unix-style newlines with a newline ending for every file
 # Indent with 4 spaces
-[php]
+[*]
+charset = utf-8
 end_of_line = lf
 indent_style = space
 indent_size = 4
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
Why do we need this?

Reply:
So when you first commented I thought "why not to trim on `md` files".. but then I saw that this was false. My bad.. so I updated this..